osquery/filesystem/windows/fileops.cpp

```diff
@@ -929,6 +933,7 @@ bool platformChmod(const std::string& path, mode_t perms) {
 
 std::vector<std::string> platformGlob(const std::string& find_path) {
   fs::path full_path(find_path);
+  boost::system::error_code ec;
```

Comment:
Push this into the loop context, replacing line 989 works with the object construction works.